ovs-p4rt/ovs_p4rt.cc

```diff
@@ -1503,6 +1509,7 @@ void ConfigFdbTableEntry(struct mac_learning_info learn_info,
       auto status_or_read_response =
           GetFdbVlanTableEntry(session.get(), learn_info, p4info);
       if (status_or_read_response.ok()) {
+        printf("Non TUNNEL: read FDB L2_FWD_TX_TABLE entry present\n");
```

Comment:
Same as above.

Reply:
this log was intentionally left for debugging purpose.